src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final int index = random.nextInt(Colors.values().length);
+
+    public String getRandomColor() {
```

Comment:
Each call to 'getRandomColor' will return the same color because the 'index' is only computed once when the 'ColorSupplier' is instantiated. To ensure a new random color is returned on each method call, move the random index generation into the 'getRandomColor' method.